script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -700,7 +700,7 @@ To be connected to it would be to 'know' the end of all things.
 The brains of higher animals posses the faculties of reception and transmission,
  but in nearly all humans those circuits are closed off to themselves.
  However, among them are those with latent potential - open circuits.
- Despite not being members of a transcendent species, these humans can still effect supernatural phenomena
+ Despite not being members of a transcendent species, those humans can still effect supernatural phenomena
```

Comment:
If something is the current subject, it's 'close-y' so I think `these` fits better than `those`

Reply:
Hmm, no for that one `those` is definitely correct. I think this is just English being weird.